pkg/common/cns-lib/vsphere/virtualcentermanager.go

```diff
@@ -52,6 +54,8 @@ type VirtualCenterManager interface {
 	IsvSANFileServicesSupported(ctx context.Context, host string) (bool, error)
 	// IsExtendVolumeSupported checks if extend volume is supported or not.
 	IsExtendVolumeSupported(ctx context.Context, host string) (bool, error)
+	// IsOnlineExtendVolumeSupported checks if online extend volume is supported or not wrt vcenter versions
+	IsOnlineExtendVolumeSupported(ctx context.Context, vCenterVersion string) bool
```

Comment:
I think we should be consistent with `IsvSANFileServicesSupported` and `IsExtendVolumeSupported` and supply hostname, instead of version as second argument.

Reply:
`IsvSANFileServicesSupported` and `IsExtendVolumeSupported` call a common function `isVsan67u3Release` which in turn retrieves the VC object to find out the client version. I retrieved the VC object in the main function itself so that it can be reused in the rest of the code if required and I am passing just the VC client version to this function which makes more sense IMO. Let me know if I should change it?

Reply:
Changed it to be consistent with other such functions.